### PR TITLE
Local GPU How-to

### PR DESCRIPTION
**Pull request by brannondorsey**
W.I.P, making a draft PR for collaboration purposes.

### CODE REVIEW

**Comment by brannondorsey** [comment]
I've finished a first draft of this. I'll give it a round of edits tomorrow, but feel free to have a look now.

File: docs/getting-started/experimental-features.md [hunk 1]
```diff
@@ -6,15 +6,34 @@ This page contains descriptions of our current experimental features and details
...
+
 - [Check out a few quick experiments](https://docs.runwayml.com/#/how-to/chaining-models-together)
 - [Check the release notes](https://runwayml.com/release-notes)
 - [Check the support page](https://support.runwayml.com/en/articles/3140662-understanding-how-chaining-works)
+
+### Local GPU
+
+?> Local GPU is available as of beta version 0.8.0
+
+If you are running Linux and have an NVIDIA graphics card that supports CUDA, you can now run models using your own GPU hardware. This feature allows you to save money by running models on your own hardware for free! See the [support page](#) for a list of prerequisites required to get local GPU working and the [tutorial](https://docs.runwayml.com/#/how-to/local-gpu) to walk through installing them yourself if you don't already have them.
```
Comment (cvalenzuela) [inline docs/getting-started/experimental-features.md]:
```suggestion
If you are running Linux and have an NVIDIA graphics card that supports CUDA, you can now run models using your own GPU hardware. See the [support page](#) for a list of prerequisites required to get local GPU working and the [tutorial](https://docs.runwayml.com/#/how-to/local-gpu) to walk through installing them yourself if you don't already have them.
```

File: docs/how-to/local-gpu.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+# Running Models Using Your Own GPU
+
+?> This document assumes the reader is loosely familiar with installing software on Linux.
+
+Local GPU is here! If you are running Linux and have an NVIDIA graphics card that supports CUDA, you can now run models locally using your own GPU. This feature allows you to save money by running models on your own hardware for free!
```
Comment (cvalenzuela) [inline docs/how-to/local-gpu.md]:
```suggestion
Local GPU is here! If you are running Runway on Linux and have an NVIDIA graphics card that supports CUDA, you can now run models locally using your own GPU.
```

File: docs/how-to/local-gpu.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+## Prerequisites
+
+In order to run models on your own GPU hardware, you must fulfill the following requirements:
+
+1. You must be using Runway on Linux (We've tested on Ubuntu 18.04, but we expect it to work with other distros and Linux flavors as well)
```
Comment (cvalenzuela) [inline docs/how-to/local-gpu.md]:
```suggestion
1. You must be using Runway on Linux (Tested on Ubuntu 18.04)
```

File: docs/how-to/local-gpu.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+1. You must have an NVIDIA GPU that supports CUDA (see a full list of supported GPUs [here](https://www.geforce.com/hardware/technology/cuda/supported-gpus))
+1. You must have a recent version of the NVIDIA Linux drivers (Drivers can be downloaded from [NVIDIA's website directly](https://www.geforce.com/drivers) or from your package manager. You must have a driver version recent enough to support CUDA 9, 9.2, and 10. See this [version compatibility matrix](https://github.com/NVIDIA/nvidia-docker/wiki/CUDA#requirements) to make sure your drivers are up to date.)
+1. You must have [NVIDIA Docker](https://github.com/NVIDIA/nvidia-docker) installed
+
+This tutorial assumes you've already completed steps 1 and 2. We'll walk you through installing the latest NVIDIA drivers for your graphics card as well as installing and configuring NVIDIA Docker so that local runway models can take advantage of your GPU hardware.
+
+In this tutorial, we'll be installing these dependencies on Ubuntu 18.04. If you are using a different distro or version, the logical steps will still apply, but the detailed commands will be different.
```
Comment (cvalenzuela) [inline docs/how-to/local-gpu.md]:
```suggestion
In this tutorial, we'll be installing these dependencies on Ubuntu 18.04. If you are using a different distribution or version, the logical steps will still apply, but the detailed commands will be different.
```

File: docs/how-to/local-gpu.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+```bash
+docker run --gpus all nvidia/cuda:9.0-base nvidia-smi
+```
+
+If all goes well you should see an ascii table similar to the one that you saw before when you ran `nvidia-smi` manually after installing the NVIDIA drivers.
+
+### Running Local GPU Models in Runway
+
+If the following steps went well, you should now be ready to run GPU models locally inside of Runway. You can chose the "LOCAL" run location and the "GPU" hardware type of any model by selecting the "Advanced Opitions" link in the workspace view. You will then be prompted to download the model, and once that's done, you can run the model locally on your own GPU!
```
Comment (cvalenzuela) [inline docs/how-to/local-gpu.md]:
```suggestion
If the following steps went well, you should now be ready to run GPU models locally inside of Runway. You can choose the "LOCAL" run location and the "GPU" hardware type of any model by selecting the "Advanced Options" link in the workspace view of Runway. You will then be prompted to download the model, and once that's done, you can run the model locally on your own GPU!
```

**Comment by brannondorsey** [comment]
Thanks for these edits @cvalenzuela!

**Pull request merged**